test/src/unit-tile-metadata.cc

```diff
@@ -555,7 +568,8 @@ struct CPPFixedTileMetadataFx {
       } else {
         CHECK_THROWS_WITH(
             frag_meta[f]->get_tile_null_count("a", tile_idx),
-            "FragmentMetadata: Trying to access metadata that's not "
+            "FragmentMetadata: Trying to access tile null count metadata "
+            "that's not "
```

Comment:
NIT: fix spacing... "that's not present"); can fit on one line. It's also in a couple other places.